apps/survey/test/survey.js

```diff
@@ -281,4 +281,25 @@ contract('Survey app', accounts => {
       })
     })
   })
+
+  context('worng supply token', () => {
```

Comment:
Small typo 😄

Reply:
'CTRL+t' is the best shortcut ever for me! Haha.